app/views/docs/databases-relationships.phtml

```diff
@@ -0,0 +1,902 @@
+        reviews: [
+            { 
+                author: 'Bob', 
+                text: 'Great movie!',
+                $permissions: [
+                    Permission.read(Role.any())
+                ]
+            },
+        ]
+    }
+)</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Flutter</h3>
+        <div class="ide" data-lang="dart" data-lang-label="Flutter SDK">
+            <pre class="line-numbers"><code class="prism language-dart" data-prism>import 'package:appwrite/appwrite.dart';
+
+final client = Client()
+    .setEndpoint('https://[HOSTNAME_OR_IP]/v1')
+    .setProject('5df5acd0d48c2');
+
+final databases = Databases(client);
+
+await databases.createDocument(
+  databaseId: 'marvel',
+  collectionId: 'movies',
+  documentId: ID.unique(),
+  data: {
+    'title': 'Spiderman',
+    'year': 2002,
+    'reviews': [
+      { 
+         'author': 'Bob', 
+         'text:' 'Great movie!',
+         '\$permissions': [
+           Permission.read(Role.any())
+         ]
+       },
+    ]
+  },
+);</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Apple</h3>
+        <div class="ide" data-lang="swift" data-lang-label="Apple SDK">
+            <pre class="line-numbers"><code class="prism language-swift" data-prism>import Appwrite
+
+let client = Client()
+    .setEndpoint("https://[HOSTNAME_OR_IP]/v1")
+    .setProject("5df5acd0d48c2")
+
+let databases = Database(client: client)
+
+databases.createDocument(
+    databaseId: "marvel",
+    collectionId: "movies",
+    documentId: ID.unique(),
+    data: [
+        "title": "Spiderman",
+        "year": 2002,
+        "reviews": [
+            [ 
+                "author": "Bob",
+                "text": "Great movie!",
+                "$permissions": [
+                    Permission.read(Role.any())
+                ]
+            ],
+        ]
+    ]
+)</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Android</h3>
+        <div class="ide" data-lang="kotlin" data-lang-label="Android SDK">
+            <pre class="line-numbers"><code class="prism language-kotlin" data-prism>import Appwrite
+
+let client = Client()
+    .setEndpoint("https://[HOSTNAME_OR_IP]/v1")
+    .setProject("5df5acd0d48c2")
+
+let databases = Database(client: client)
+
+databases.createDocument(
+    databaseId: "marvel",
+    collectionId: "movies",
+    documentId: ID.unique(),
+    data: [
+        "title": "Spiderman",
+        "year": 2002,
+        "reviews": [
+            [ 
+                "author": "Bob",
+                "text": "Great movie!",
+                "$permissions": [
+                    Permission.read(Role.any())
+                ]
+            ],
+        ]
+    ]
+)</code></pre>
+        </div>
+    </li>
+</ul>
+
+<p>When creating the parent document and linking to a child document, or updating a relationship you must have permission to access both documents. If the user does not have read permission, an exception will be thrown.</p>
```

Comment:
> an exception will be thrown.

This is true for create, update, and delete operations, but you will get an empty document for read operations.